### ISSUE TITLE
[Device Code lookup] Accessible name for `Copy Test performed code for [device name]` is not appropriate in the case of multiplex devices

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#5858]
## Area or Component

Device Code Lookup 

## Quick Summary

The test performed code copy button  is not descriptive enough for multiplex devices. It should also include the disease in the name. Currently it reads `Copy Test performed code for Abbott Alinity M` for each of the diseases.

## Accessibility Violation

https://docs.deque.com/issue-help/1.0.0/en/aria-name-missing-incorrect

## Action Requested

Add the disease name in the aria label 

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/17c28123b9674ffbe4eef6b47ebcdf960d32d87e/frontend/src/app/uploads/DeviceLookup/DeviceDetails.tsx#L116-L119
